dev/NavigationView/NavigationViewItemBase.cpp

```diff
@@ -35,25 +35,19 @@ void NavigationViewItemBase::Position(NavigationViewListPosition value)
     if (m_position != value)
     {
         m_position = value;
-        OnNavigationViewListPositionChanged();
+        OnNavigationViewPositionChanged();
     }
 }
 
 winrt::NavigationView NavigationViewItemBase::GetNavigationView()
 {
-    //Because of Overflow popup, we can't get NavigationView by SharedHelpers::GetAncestorOfType
```

Comment:
>//Because of Overflow popup, we can't get NavigationView by SharedHelpers::GetAncestorOfType [](start = 4, length = 92)

Is this comment no longer true?

Reply:
I just pushed an update deleting the SharedHelpers::GetAncestorOfType call because it is not needed anymore. So comment is still true but no longer relevant here.